.cirrus.yml

```diff
@@ -0,0 +1,46 @@
+freebsd_instance:
+  image_family: freebsd-12-1-snap
+
+test_task:
+  name: Tests (FreeBSD)
+  env:
+    matrix:
+      - PYTHON: python2.7
+      - PYTHON: python3.5
+      - PYTHON: python3.6
+      - PYTHON: python3.7
+      - PYTHON: python3.8
+  python_script:
+    - PYPACKAGE=$(printf '%s' $PYTHON | tr -d '.')
+    - SQLPACKAGE=$(printf '%s-sqlite3' $PYPACKAGE | sed 's/thon//')
+    - pkg install -y git-lite $PYPACKAGE $SQLPACKAGE
```

Comment:
Are these required?

Reply:
Not sure what you mean by are these required -- do we need `git` and `sqlite` bindings for the test suite to run? Yes, we do. Do we need to do string transforms to get the right package name? If we want pretty/readable test names without many environmental variables, yes we do. Take a look at the commit history to see what it looks like without the string manipulation.

Reply:
Apologies. I should have been a bit more clear and probably more awake. I was curious about the sqllite package. Didn't occor this was being used by pytest. Ignore my comment.